src/main/java/org/terasology/logic/ingame/towers/TowerComponent.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.ingame.towers;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.network.Replicate;
+import org.terasology.world.block.ForceBlockActive;
+
+@ForceBlockActive
+public class TowerComponent implements Component {
```

Comment:
Again, JavaDoc is highly appreciated (but can be added in a later PR).

Reply:
Same as the previous comment.